Aflores plymouth (#234)
osisoft/Edge-Data-Store-Docs: content/opc-ua/opc-ua-overview.md

## Overview

> The OPC UA EDS adapter is a component of Edge Data Store that transfers time-series data from an OPC UA capable device to EDS. OPC Unified Architecture (OPC UA) is a machine to machine communication protocol for industrial automation developed by the OPC Foundation. The OPC UA EDS adapter can connect to any device that uses the OPC UA communication protocol, and uses subscriptions to reduce the amount of data transferred by only reading data changes and events.

Reviewer: ```suggestion
The OPC UA EDS adapter is a component of Edge Data Store that transfers time-series data from an OPC UA capable device to EDS. OPC Unified Architecture (OPC UA) is a machine to machine communication protocol for industrial automation developed by the OPC Foundation. The OPC UA EDS adapter can connect to any device that uses the OPC UA communication protocol and uses subscriptions to reduce the amount of data transferred by only reading data changes and events.
```

The following diagram depicts the data flow for a single instance of OPC UA EDS adapter instance:

![OPC UA EDS](https://osisoft.github.io/Edge-Data-Store-Docs/content/images/OPCUAConfiguration.jpg "OPC UA Configuration")

> The adapter instance polls the OPC UA device and then collects data from the device. The adapter then sends the data to the storage component where it is held until it can be egressed to permanent storage in PI Server or AVEVA Data Hub. The adapter instance can be configured from the device where EDS is installed, and EDS collects health information about the adapter that can be egressed.

Reviewer: ```suggestion
The adapter instance polls the OPC UA device and then collects data from the device. The adapter then sends the data to the storage component where it is held until it can be egressed to permanent storage in PI Server or AVEVA Data Hub. You can configure the adapter instance from the device where EDS is installed. EDS also collects health information about the adapter that can be egressed.
```

  
  * Data selection configuration - specifies what data is collected from the device and how it is identified. Each adapter component instance requires a data selection configuration.

> With these configurations completed, the OPC UA EDS adapter instance collects data from the specified device and sends it to EDS, where it is stored locally until it can be sent to a PI System or AVEVA Data Hub for long-term storage and analysis.

Reviewer: ```suggestion
With these configurations completed, the OPC UA EDS adapter instance collects data from the specified device and sends it to EDS where it is stored locally until it can be sent to a PI System or AVEVA Data Hub for long-term storage and analysis.
```
